site/docs/patterns/app-header.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: App header
+layout: DetailPattern
+aliases:
+  - /salt/patterns/app-header
+data:
+  resources:
+    [
+      { href: "https://www.saltdesignsystem.com/salt/components/navigation-item/examples", label: "Navigation item" },
+    ]
+---
+
+The app header is a consistently available element that sits on top of every application interface. It contains a brand logo or mark and application-wide functionality, including navigation and utility actions. This pattern introduces guidelines to make the app header consistent and usable. 
+
+  <Diagram
+    src="/img/patterns/app-header/app-header.png"
+    alt="App header example"
+    border
+  />
+
+## When to use it
+
+The app header serves as the primary element for users to orient themselves within the UI. It should be globally persistent, enabling easy navigation and access to utility functions.
+
+## How to build it
+
+### Anatomy
+
+The app header has three functional areas:
+
+1. **Branding:** Displays a brand logo or mark followed by the application name in text form. This should always link to the product’s home page.
+2. **Navigation:** Serves as the container for the primary navigation. Its inclusion is optional if you have single-page applications or use a persistent side navigation.
+3. **Utilities:** Provides access to persistent application-wide utilities, such as external links, search, notifications, help, account details and settings.
+
+  <Diagram
+    src="/img/patterns/app-header/anatomy.png"
+    alt="Header functional areas—branding, navigation and utilities."
+    border
+  />
+
+### Desktop layout construction
+
+  <Diagram
+    src="/img/patterns/app-header/desktop.png"
+    alt="Example showing header’s desktop layout construction."
+    border
+  />
+
+### Mobile layout construction
+
+  <Diagram
+    src="/img/patterns/app-header/mobile.png"
+    alt="Example showing header’s mobile layout construction."
+    border
+  />
+
+## Variants
+
+### Header only
+
+In this layout, the app header is globally persistent, and you can use it as standalone navigation for a small number of sections that don’t require secondary navigation. Although this layout provides more horizontal space for content, it has limited space for navigation items, and combined navigation may be required to navigate complex hierarchies.
+
+  <Diagram
+    src="/img/patterns/app-header/header-only.png"
+    alt="Example of the header-only variant."
+    border
+  />
+
+### Header with vertical navigation
```

Comment:
More of a personal opinion, this section seems very navigation specific

Reply:
Agree with you Josh. That is why we kept this section as a general overview to explain how the app header works when using vertical navigation together. We can probably revisit both, navigation and app header patterns, in the future to remove this and avoid information overlaps.